packages/dart/common/lib/src/environment.dart

```diff
@@ -28,6 +28,9 @@ class Environment {
   /// The URL of the IoT service.
   final String iotUrl;
 
+  /// The URL of the Vault.
+  final String vaultUrl;
```

Comment:
Name is a bit confusing. This is not the Affinidi Vault url, right? this seems to be an elements url for vault api services